README.md

```diff
@@ -44,11 +44,12 @@ Full path to where Who's on First data is located (note: the included [downloade
 * Required: no
 * Default: ``
 
-Set to a WOF ID or array of IDs to import data only for descendants of those records, rather than the entire planet.
+Set to a ISO Country CODE or array of CODEs to import data only for descendants of those records, rather than the entire planet. Example: `fr` or `['fr', 'ch']`
 
-You can use the [Who's on First Spelunker](https://spelunker.whosonfirst.org) or the `source_id` field from any WOF result of a Pelias query to determine these values.
+For United States you can provide CODE and SUBDIVISION combination. Example: `us-ca` or `['us-ca', 'us-il']`
+
+Check out available options here https://dist.whosonfirst.org/sqlite/.
```

Comment:
Thank you @pixeldublu I have been waiting for this feature. 

The thing about `importPlace` is that it also allows you to import individual regions or cities (basically anything with a WOF id).
This PR seems to break this feature.

Reply:
I see :) Ok no worries. Will move the iso codes into a new option and leave wofid importplace as before.

Reply:
@bboure 
Things complicated over the weekend. WOF Disabled downloads so only geocode.earth is an option now. And only sqlite. 

I did some changes... seems to work... but its rudimentary. So i am interested in some suggestions on how to proceed about this.